lua/autorun/photon/cl_emv_meta.lua

```diff
@@ -1006,4 +1006,385 @@ function EMVU:MakeEMV( emv, name )
+								v.ManualSiren:ChangeVolume(math.Clamp(1 + (distBehind/fadeDist), 0.05, 1), updateRate)
+							end
+						end
+
+						if math.abs(doppler) > 1 then
+								v.ManualSiren:ChangePitch(math.Clamp(100 + doppler, 50, 150), updateRate)
+							elseif v.ManualSiren:GetPitch() ~= 100 then
+								v.ManualSiren:ChangePitch(100, updateRate)
+						end
+					else
+						if v.ManualSiren:GetPitch() ~= 100 then
+							v.ManualSiren:ChangePitch(100)
+						end
+						if v.ManualSiren:GetVolume() ~= thirdPersonVolume and v:GetThirdPersonMode() then
+							v.ManualSiren:ChangeVolume(thirdPersonVolume, updateRate)
+						elseif v.ManualSiren:GetVolume() ~= interiorVolume and !v:GetThirdPersonMode() then
+							v.ManualSiren:ChangeVolume(interiorVolume, updateRate)
+						end
+					end
+				end
+
+				if v.Horn then
+					if (IsValid(v:GetDriver()) and v:GetDriver() ~= ply) or !IsValid(v:GetDriver()) then
+						local distBehind = v:WorldToLocal(ply:GetPos())[2]
+						
+						if IsValid(ply:GetVehicle()) then
+							if ply:GetVehicle():GetParent() == v then
+								if v.Horn:GetVolume() ~= thirdPersonVolume and ply:GetVehicle():GetThirdPersonMode() then
+									v.Horn:ChangeVolume(thirdPersonVolume, updateRate)
+								elseif v.Horn:GetVolume() ~= interiorVolume and !ply:GetVehicle():GetThirdPersonMode() then
+									v.Horn:ChangeVolume(interiorVolume, updateRate)
+								end
+							end
+						else
+							if v.Horn:GetVolume() ~= 1 and distBehind > 0 then
+								v.Horn:ChangeVolume(1, updateRate)
+							elseif distBehind < 0 then
+								v.Horn:ChangeVolume(math.Clamp(1 + (distBehind/fadeDist), 0.05, 1), updateRate)
+							end
+						end
+
+						if math.abs(doppler) > 1 then
+								v.Horn:ChangePitch(math.Clamp(100 + doppler, 50, 150), updateRate)
+							elseif v.Horn:GetPitch() ~= 100 then
+								v.Horn:ChangePitch(100, updateRate)
+						end
+					else
+						if v.Horn:GetPitch() ~= 100 then
+							v.Horn:ChangePitch(100)
+						end
+						if v.Horn:GetVolume() ~= thirdPersonVolume and v:GetThirdPersonMode() then
+							v.Horn:ChangeVolume(thirdPersonVolume, updateRate)
+						elseif v.Horn:GetVolume() ~= interiorVolume and !v:GetThirdPersonMode() then
+							v.Horn:ChangeVolume(interiorVolume, updateRate)
+						end
+					end
+				end
+			end
+		end
+		NextDoppler = CurTime() + updateRate
+	end
+end)
+
+net.Receive("Photon.ELS_PlaySiren", function()
+	local ent = net.ReadEntity()
+	if ent:IsValid() and !IsValid(ent.Siren) then
+		local sound = net.ReadString()
+		local volume = net.ReadFloat()
+		ent.Siren = CreateSound(ent, sound)
+		ent.Siren:SetSoundLevel( volume * 1.8 )
+		if ent:GetThirdPersonMode() then
+			ent.Siren:PlayEx(thirdPersonVolume, 100)
+		elseif !ent:GetThirdPersonMode() then
+			ent.Siren:PlayEx(interiorVolume, 100)
+		else
+			ent.Siren:PlayEx(1, 100)
+		end
+		--ent.Siren:ChangeVolume(0)
+		--ent.Siren:SetDSP(1)
+		ent:CallOnRemove("StopSiren"..ent:EntIndex(), function(ent)
```

Comment:
CallOnRemove is localised to the entity already, no need to append the entindex.